test/parse.jl

```diff
@@ -99,9 +99,6 @@ macro test999_str(args...); args; end
 # issue #10994
 @test parse("1 + #= \0 =# 2") == :(1 + 2)
 
-# issue #10985
-@test expand(:(f(::Int...) = 1)).head == :method
```

Comment:
It's a 2-statement block that adds a method and then returns `f`. I guess I might revisit whether this is necessary.